src/Duo.Clients.Wpf/Messaging/ApriManutenzioneTrattamentoMessage.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Duo.Domain.ViewModels.Trattamenti;
+
+namespace Duo.Clients.Wpf.Messaging
+{
+    class ApriManutenzioneTrattamentoMessage
+    {
+        public TrattamentoView Trattamento { get; set; }
```

Comment:
L'idea è di passargli il trattamento in caso di modifica (codice che non avevo ancora scritto)
        public void ModificaTrattamento()
        {
            this.broker.Broadcast(this, new Messaging.ApriManutenzioneTrattamentoMessage
            {
                Trattamento = TrattamentoSelezionato
            });
        }